packages/lab/src/slider/Slider.tsx

```diff
@@ -75,99 +69,59 @@ export const Slider = forwardRef<HTMLDivElement, SliderProps>(function Slider(
     window: targetWindow,
   });
 
-  const trackRef = useRef<HTMLDivElement>(null);
-
   const [value, setValue] = useControlled<SliderValue>({
-    controlled: valueProp,
+    controlled: parseValueProp(valueProp, min, max),
     default: defaultValue,
```

Comment:
Why is only controlled "parsed"? I'm not sure if changing the passed in value is the correct thing here?